BUILD

```diff
@@ -5,10 +5,17 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_bundle")
 container_bundle(
     name = "all",
     images = {
-        "gcr.io/{PROJECT_ID}/static:nonroot": "//base:static-nonroot",
-        "gcr.io/{PROJECT_ID}/base:nonroot": "//base:base-nonroot",
         "gcr.io/{PROJECT_ID}/base:debug": "//base:debug",
+        "gcr.io/{PROJECT_ID}/base:debug-debian10": "//base:debug-debian10",
```

Comment:
Nit: it might be better to suffix all old images with `-debian9` (but keep non-suffixed labels). In the future the non-suffixed version can be deprecated.

Reply:
I would prefer to leave the current tagging unchanged until debian10 moves out of experimental.